docs/migration-guide-1.24-2.0.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Migration Guide 1.24-2.0
+---
+
+Once you upgrade to Kyma 2.0, perform the manual steps described in the Migration Guide.
+
+## Security
+
+### Native Kubernetes authentication in Kyma
+
+Kyma 2.0 simplifies things and switches to the native Kubernetes authentication. As a result, we no longer support the following authentication and authorization components:
+
```

Comment:
Please add section WARNING:
These components are removed by migration script. If you want to keep them please modify the script before you execute it accordingly.

Reply:
Added at the end of the section